README.md

```diff
@@ -20,12 +23,13 @@ Initial plan is to validate containers/images/dockerfiles against different ecos
 
 ### Installing via `pip`
 
-We haven't released colin on PyPI yet, in the meantime, you can install it directly from git:
 
 ```bash
-$ pip3 install --user git+https://github.com/user-cont/colin.git
+$ pip3 install --user colin
 ```
 
+> You may need to install also rpm packages `python3-devel` and `python3-pyxattr`.
```

Comment:
If you have pyxattr, I believe you don't need python-devel then (b/c there is nothing to compile, right?) Therefore:
1. Could we make sure that we just need pyxattr and not -devel as well?
2. Since we try to make the tool generic, let's word it like this: *If you are on Fedora distribution, please install python3-pyxattr so you don't have compile yourself when getting it from PyPI*.